modules/context/repo.go

```diff
@@ -1087,6 +1087,9 @@ func (ctx *Context) IssueTemplatesErrorsFromDefaultBranch() ([]*api.IssueTemplat
 			if it, err := template.UnmarshalFromEntry(entry, dirName); err != nil {
 				invalidFiles[fullName] = err
 			} else {
+				if !strings.HasPrefix(it.Ref, "refs/") { // Assume that the ref intended is always a branch - for tags users should use refs/tags/<ref>
```

Comment:
Why not
```suggestion
				if !strings.HasPrefix(it.Ref, git.BranchPrefix) { // Assume that the ref intended is always a branch - for tags users should use refs/tags/<ref>
```
?

Reply:
It's not the same. `refs/heads` means branch not `refs`.

Reply:
yes. But have a look one line below the suggestion to see why I'm somewhat against the current implementation.

Reply:
Okay, yeah, I can see now that it won't work when trying to use tags.
Perhaps we should add a new Prefix: `const git.RefPrefix = "refs/"` and adapt the other prefixes to `RefPrefix + rest`

Reply:
"refs/" is used in these two places only.

It seems OTT to create a const and frankly more confusing for that. 

The correct solution if we want to be strict is to create a series of types that would ensure that this was done correctly however that is a bigger job than this PR, (which I would note has itself already taken 3 months to get review.)

Reply:
Yes